.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+## Changes description 🧑‍💻
+<!--- Describe your changes in detail -->
+
+## Context 🤔
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it is solving an issue... How can it be reproduced in order to compare between both behaviors? -->
+
+## Checklist ✅
+<!--- Feel free to add other steps if needed -->
+- [ ] Check your code additions will fail neither code linting checks.
```

Comment:
This could be reworded to something simpler. Or even remove since we enforce lint checks.